src/components/MaModal/MaModal.scss

```
@@ -2,41 +2,37 @@
   position: fixed;
   top: 0;
   left: 0;
-  right: 0;
-  bottom: 0;
-
-  background-color: rgba(0, 0, 0, 0.25);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: opacity 0.4s ease;
+  background-color: get-alpha-color(black, 0.5);
+  width: 100%;
+  height: 100%;
 }
 
 .modal {
-  position: fixed;
-
-  // centering
-  left: 50%;
-  top: 50%;
-  transform: translate(-50%, -50%);
+  transition: transform 0.4s ease;
+  border-radius: 0.25rem;
+  box-shadow: 0 rem(2px) rem(8px) get-alpha-color(black, 0.67);
```

Do we want borders that have a variable width depending on user settings of base font size? 😅 

```suggestion
  border-radius: 4px;
  box-shadow: 0 2px 8px get-alpha-color(black, 0.67);
```

(Until we have some border/shadow variables from the design system, I guess! But that's beyond the scope of this feature)